lib/driver.js

```diff
@@ -18,14 +18,25 @@ const helpers = Object.assign({}, uiautomator2Helpers, androidHelpers);
 
 // The range of ports we can use on the system for communicating to the
 // UiAutomator2 HTTP server on the device
-const SYSTEM_PORT_RANGE = [8200, 8299];
-// The guard is needed to avoid dynamic port allocation conflicts
+const DEVICE_PORT_RANGE = [8200, 8299];
+// The range of ports we can use on the system for communicating to the
+// UiAutomator2 MJPEG server on the device
+const MJPEG_SERVER_PORT_RANGE = [9200, 9299];
+
+// The guard is needed to avoid dynamic system port allocation conflicts for
+// parallel driver sessions
+const DEVICE_PORT_ALLOCATION_GUARD = new AsyncLock();
+// The guard is needed to avoid dynamic MJPEG server  port allocation conflicts
```

Comment:
nit: duplicated space

Reply:
Good catch. This will be resolved w/ the upcoming update(s).